File: 7.js
// Sigma n^n/n! (LOOPS)

function Factorial(range)
{
    let pi = 1;
    for (let i = 1; i <= range; i++)
    {
        pi *= i;
    }
    return pi;
}
function Summation(range)
{
    let sum = 0;

    for (let i = 1; i <= range; i++)
    {
        sum += i ** i / Factorial(i);
    }
    return sum;
}
console.log(Summation(10));

File: 7.c
// Sigma n^n/n! (LOOPS)
#include <stdio.h>
#include <math.h>
int Factorial(int range);
int Summation(int range);
int main()
{
    printf("%d\n", Summation(10));
    return 0;
}
int Factorial(int range)
{
    int pi = 1;

    for (int i = 1; i <= range; i++)
    {
        pi *= i;
    }
    return pi;
}
int Summation(int range)
{
    int sum = 0;
    for (int i = 1; i <= range; i++)
    {
        sum += pow(i, i) / Factorial(i);
    }
    return sum;
}